src/wix/WixToolset.Core/CommandLine/CommandLine.cs

```diff
@@ -56,7 +56,7 @@ public ICommandLineCommand ParseStandardCommandLine(ICommandLineArguments argume
             if (command.ShowLogo)
             {
                 var branding = this.ServiceProvider.GetService<IWixBranding>();
-                Console.WriteLine(branding.ReplacePlaceholders("[AssemblyProduct] [AssemblyDescription] version [FileVersion]"));
+                Console.WriteLine(branding.ReplacePlaceholders("[AssemblyProduct] [AssemblyDescription] version [AssemblyVersion]"));
```

Comment:
I probably should have expounded upon this comment a bit more:

> Better to update WixBranding to add a new ProductVersion replacement and use that. Then this won't be a special casing using ThisAssembly.

Rather than changing the behavior of `AssemblyVersion` add a new `ProductVersion` replacement to WixBranding. Use `ProductVersion` here instead of `FileVersion` and (now) `AssemblyVersion`.